core/src/main/resources/defaults/blueprints/cdp-data-mart-active.bp

```diff
@@ -0,0 +1,226 @@
+            }, {
+              "name" : "fs_data_dirs",
+              "value" : "/hadoopfs/fs1/master"
+            } ]
+          },
+          {
+            "refName": "kudu-TSERVER-BASE",
+            "roleType": "KUDU_TSERVER",
+            "base": true,
+            "configs" : [ {
+              "name" : "fs_wal_dir",
+              "value" : "/hadoopfs/fs1/tserver"
+            }, {
+              "name" : "fs_data_dirs",
+              "value" : "/hadoopfs/fs1/tserver"
+            } ]
+          }
+        ]
+      },
+      {
+        "refName": "yarn",
+        "serviceType": "YARN",
+        "roleConfigGroups": [
+          {
+            "refName": "yarn-RESOURCEMANAGER-BASE",
+            "roleType": "RESOURCEMANAGER",
+            "base": true
+          },
+          {
+            "refName": "yarn-NODEMANAGER-BASE",
+            "roleType": "NODEMANAGER",
+            "base": true
+          },
+          {
+            "refName": "yarn-JOBHISTORY-BASE",
+            "roleType": "JOBHISTORY",
+            "base": true
+          }
+        ]
+      },
+      {
+        "refName": "spark_on_yarn",
+        "serviceType": "SPARK_ON_YARN",
+        "roleConfigGroups": [
+          {
+            "refName": "spark_on_yarn-SPARK_YARN_HISTORY_SERVER-BASE",
+            "roleType": "SPARK_YARN_HISTORY_SERVER",
+            "base": true
+          },
+          {
+            "refName": "spark_on_yarn-GATEWAY-BASE",
+            "roleType": "GATEWAY",
+            "base": true
+          }
+        ]
+      },
+      {
+        "refName": "impala",
+        "serviceType": "IMPALA",
+        "serviceConfigs" : [ {
+          "name" : "impala_cmd_args_safety_valve",
+          "value" : "--cache_s3_file_handles=true"
+        } ],
+        "roleConfigGroups": [
+          {
+            "refName": "impala-IMPALAD-COORDINATOR",
+            "roleType": "IMPALAD",
+            "configs" : [ {
+              "name" : "impalad_specialization",
+              "value" : "COORDINATOR_ONLY"
+            }, {
+              "name" : "impala_hdfs_site_conf_safety_valve",
+              "value" : "<property><name>fs.s3a.block.size</name><value>268435456</value></property><property><name>fs.s3a.experimental.input.fadvise</name><value>RANDOM</value></property><property><name>fs.s3a.fast.upload</name><value>true</value></property>"
+            } ],
+            "base": false
+          },
+          {
+            "refName": "impala-IMPALAD-EXECUTOR",
+            "roleType": "IMPALAD",
+            "configs" : [ {
+              "name" : "impalad_specialization",
+              "value" : "EXECUTOR_ONLY"
+            }, {
+              "name" : "impala_hdfs_site_conf_safety_valve",
+              "value" : "<property><name>fs.s3a.block.size</name><value>268435456</value></property><property><name>fs.s3a.experimental.input.fadvise</name><value>RANDOM</value></property><property><name>fs.s3a.fast.upload</name><value>true</value></property>"
```

Comment:
with this property I think we are tightly coupled with AWS do we really need this ? I think the blueprint should not depend on cloud providers.
ccc @akanto @keyki

Reply:
These configurations are already set in `cdp-data-mart.bp`. I think they were copied from there.

Reply:
Yeah, I copied these straight from `cdp-data-mart.bp` without thinking much about it. Guess this is a question for @drorke?

Reply:
Some of the S3 related settings will be removed after some additional work to improve the defaults in the Impala code (likely to land in M8) but some S3 config tuning is likely to remain in the blueprint.  I think these are fairly benign because they'll essentially be ignored when running on Azure or in any other environment where the S3A file system type isn't used (I'm not aware of any scenario where having these in the safety valve would cause an error).

Another option would be to add them via code in a CB configprovider, assuming the configprovider had a way to know what cloud provider it was running on.